src/CMakeLists.txt

```diff
@@ -12,3 +12,6 @@ target_link_libraries(
           docopt::docopt
           fmt::fmt
           spdlog::spdlog)
+
+install(TARGETS intro)
```

Comment:
This works because it is only an executable. To really test a packaging solution, you need to run the test suit of `project_options`, where the packaging is tested for a static library, header-only library and executable. It also tests the call site using `find_package(package CONFIG REQUIRED)` and `target_link_libraries(another_project PRIVATE targets)`

https://github.com/cpp-best-practices/project_options/blob/edc28d8c50b7b87eeedcf5547a5f2b655f47cfcc/Taskfile.yml#L15

Reply:
This is actually what you need if you want to do this manually.

https://github.com/fmtlib/fmt/blob/d8e1dd4ab291862498e40542b74c91124c5aa06e/CMakeLists.txt#L288-L352

`package_project()` does this in one line of Code.